### DISCUSSION TITLE
Snapshot Quality/Resolution

### DISCUSSION THREAD

**Discussion by broyuken** [tsightler/ring-mqtt#383] [Support Q&A] [answered]
Is there a way to get the snapshot cameras in a higher resolution? They seem to be pretty low quality, I’m not sure if that’s how they come from ring or if it’s something coded in ring-mqtt.

**Comment by tsightler** [comment, marked as answer]
The Ring snapshots are mostly designed for low bandwidth and power usage for use as thumbnail previews in the app/web UI or with notifications on your device, they are definitely not full resolution.  I believe they are 640x360, so quite low, standard definition resolution, and ring-mqtt does not modify them in any way from how they are received.  Literally just pulls them straight from the snapshot URL in the API and pushes them out via MQTT with zero modification.

**Comment by broyuken** [comment]
Unfortunately that’s what I figured, but thank you for confirming.
